server/src/main/java/org/elasticsearch/common/settings/Setting.java

```diff
@@ -1266,6 +1257,51 @@ private static String arrayToParsableString(List<String> array) {
         }
     }
 
+    private static class ListSetting<T> extends Setting<List<T>> {
+
+        private final Function<Settings, List<String>> defaultStringValue;
+
+        private ListSetting(
+                final String key,
+                final @Nullable Setting<List<T>> fallbackSetting,
```

Comment:
Fallback settings were not being pushed down here, so `ListSetting`s did not know until this change that they could have fallback settings.